tests/xgboost/test_xgboost_autolog.py

```diff
@@ -192,14 +187,13 @@ def test_xgb_autolog_sklearn():
 
     with mlflow.start_run() as run:
         model.fit(X, y)
-        model_uri = mlflow.get_artifact_uri("model")
 
     client = MlflowClient()
     run = client.get_run(run.info.run_id)
     assert run.data.metrics.items() <= params.items()
     artifacts = {x.path for x in client.list_artifacts(run.info.run_id)}
     assert artifacts >= {"feature_importance_weight.png", "feature_importance_weight.json"}
-    loaded_model = mlflow.xgboost.load_model(model_uri)
+    loaded_model = mlflow.xgboost.load_model(f"runs:/{run.info.run_id}/model")
```

Comment:
Could we use last_logged_model to fetch model_uri

Reply:
We could, do we always want to use `mlflow.last_logged_model().model_uri` instead of `runs:/{run.info.run_id}/{model name}`? The former way should also continue to work in mlflow3.

Reply:
No clear winner in this case. Using `runs:/` is fine.